tests/cli/test_rasa_train.py

```diff
@@ -419,7 +422,8 @@ def test_train_core_help(run: Callable[..., RunResult]):
                        [--augmentation AUGMENTATION] [--debug-plots] [--force]
                        [--fixed-model-name FIXED_MODEL_NAME]
                        [--percentages [PERCENTAGES [PERCENTAGES ...]]]
-                       [--runs RUNS]"""
+                       [--runs RUNS] [--finetune [FINETUNE]]
+                       [--epoch-fraction EPOCH_FRACTION]"""
```

Comment:
There are no tests for the command itself, right? I think it makes sense to add them.
(something with `run_in_simple_project("train", "--finetune"`)

Reply:
I've done a basic cli test with the command (which just tests the arg is actually passed down). And I've changed `test_model_finetuning` to not mock the train methods and actually run the second training.